src/extensions/verify.py

```diff
@@ -0,0 +1,76 @@
+import arc
+import hikari
+
+from src.config import CHANNEL_IDS, ROLE_IDS
+from src.hooks import restrict_to_channels, restrict_to_roles
+
+plugin = arc.GatewayPlugin(name="Verify")
+
+
+@plugin.include
+@arc.with_hook(
+    restrict_to_roles(role_ids=[ROLE_IDS["admins"]]),
+)
+@arc.with_hook(restrict_to_channels(channel_ids=[CHANNEL_IDS["bot-private"]]))
+@arc.slash_command(
+    "verify",
+    "Verify a Redbrick account.",
+)
+async def verify_command(
+    ctx: arc.GatewayContext,
+    username: arc.Option[
+        str,
+        arc.StrParams("Redbrick username.", min_length=3, max_length=8),
+    ],
+    user: arc.Option[
+        hikari.User,
+        arc.UserParams("The Discord user to verify"),
+    ],
+) -> None:
+    """Verify a Discord member against a Redbrick account."""
+
+    if ctx.guild_id is None:
+        await ctx.respond(
+            "This command can only be used in a guild.",
+            flags=hikari.MessageFlag.EPHEMERAL,
+        )
+        return
```

Comment:
This is unnecessary as commands are disabled in DMs entirely (see [here](https://github.com/redbrick/blockbot/blob/main/src/bot.py#L22)). Instead, you can `assert ctx.guild_id is not None`